### PR TITLE
feat(CDH-457): allow psr/http-message ^2.0.0;

### PR DESCRIPTION
**Pull request by nikolatrajkovic994**

### CODE REVIEW

**Review by ig0rN — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
     "require": {
         "php": "^8.0",
         "ext-json": "*",
-        "psr/http-message": "^1.0.1",
-        "psr/http-client": "^1.0.1"
+        "psr/http-client": "^1.0.1",
+        "psr/http-message": "^1.0.1 || ^2.0.0"
```
Comment (komando82) [inline composer.json:26]: I want to check something ...

**Review by komando82 — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (komando82) [reply composer.json:26]: checked all good, sorry...

**Review by komando82 — APPROVED** [review]

**Comment by nikolatrajkovic994** [comment]
Hi @ig0rN, 
Just to double check, next target release is `v2.1.0`?

**Comment by ig0rN** [comment]
> Hi @ig0rN,
> Just to double check, next target release is v2.1.0?

Yup

**Pull request merged**